api/src/routes/schedule.ts

```diff
@@ -133,8 +133,30 @@ export class ScheduleRouting extends Routing {
         return res.status(201).json(schedule);
     }
 
-    @Auth.authorization({ superStudent: true })
+    @Auth.authorization({ student: true })
     async updateOne(req: CustomRequest, res: express.Response) {
+        if (req.user?.student) {
```

Comment:
Ik zou hierbij ook nagaan dat de gebruiker geen superstudent of admin is. Want een gebruiker kan meerdere rollen hebben en hierdoor zou die maar beperkte toegang hebben terwijl die bv ook superstudent is.
``` (javascript)
if (req.user?.student && !req.user?.super_student && !req.user?.admin)
```